CHANGELOG

```diff
@@ -1,3 +1,9 @@
+v3.7.15 (XXXX-XX-XX)
+--------------------
+
+* Append physical compaction of log collection to every Raft log
+  compaction (BTS-542).
```

Comment:
We should also mention here that we are now limiting the RocksDB block cache size and write buffer size for agent instances by default.